pkg/coreapi/graph/resolvers/runs_v2.go

```diff
@@ -94,6 +94,7 @@ func (r *queryResolver) Runs(ctx context.Context, num int, cur *string, order []
 			IsBatch:        r.IsBatch,
 			BatchCreatedAt: batchTime,
 			CronSchedule:   r.CronSchedule,
+			HasAi:          r.HasAI,
```

Comment:
generated code turns my has_ai field into `hasAi` and not `hasAI` which is a bit unfortunate. Can I override this?

Reply:
Aha - the Go conventions battling GraphQL codegen! I think you could change the gqlgen file but I wouldn't worry.